tests/manage/mcg/test_bucket_deletion.py

```diff
@@ -1,51 +1,215 @@
 import logging
 
+import timeit
+import botocore
 import pytest
 
 from ocs_ci.framework.pytest_customization.marks import (
-    tier1, noobaa_cli_required, aws_platform_required, acceptance,
-    filter_insecure_request_warning
+    tier1, tier2, tier3, noobaa_cli_required, aws_platform_required,
+    acceptance, filter_insecure_request_warning
 )
+from ocs_ci.ocs.exceptions import CommandFailed
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.resources.mcg_bucket import S3Bucket, OCBucket, CLIBucket
+from ocs_ci.utility.utils import run_mcg_cmd
+from tests.helpers import create_unique_resource_name
+from tests.manage.mcg import helpers
 
 logger = logging.getLogger(__name__)
 
 
 @filter_insecure_request_warning
 @aws_platform_required
-@acceptance
-@tier1
 class TestBucketDeletion:
     """
     Test bucket Creation Deletion of buckets
     """
-    @pytest.mark.polarion_id("OCS-1299")
-    def test_s3_bucket_delete(self, mcg_obj, bucket_factory):
+    @pytest.mark.parametrize(
+        argnames="amount,interface",
+        argvalues=[
+            pytest.param(
+                *[3, 'S3'],
+                marks=[pytest.mark.polarion_id("OCS-1299"), tier1, acceptance]
+            ),
+            pytest.param(
+                *[3, 'CLI'],
+                marks=[tier1, acceptance, noobaa_cli_required,
+                       pytest.mark.polarion_id("OCS-1299")]
```

Comment:
multiple cases are marked with OCS-1299. Does it reflects the polarion test cases?
The same for 1400

Reply:
1299 was the original case here cover both OBCs and s3 buckets, as I parametrize it with add more new cases I left it unchanged.
1400 was a new case I created though, and it's also for both OBCs and s3 buckets.

Reply:
updated all with 1:1 mapping